draft-ZIP-0227.rst

```diff
@@ -0,0 +1,184 @@
+::
+
+  ZIP: 227
+  Title: Issuance of Zcash Shielded Assets
+  Owners: Daniel Benarroch <daniel@qed-it.com>
+          Pablo Kogan <pablo@qed-it.com>
+          Aurelien Nicolas <aurel@qed-it.com>
+  Credits: Daira Hopwood
+           Jack Grigg
+           Shahaf Nacshon
+           Vivek Arte
+  Status: Draft
+  Category: Consensus
+  Created: 2022-05-01
+  License: MIT
+  Discussions-To: <https://github.com/zcash/zips/issues/618>
+
+
+Terminology
+===========
+
+The key word "MUST" in this document is to be interpreted as described in RFC 2119 [#RFC2119]_.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
+
+The term "Orchard" in this document is to be interpreted as described in ZIP 224 [#zip-0224]_.
+
+We define the following additional terms:
+
+- Asset: A type of note that can be transferred on the Zcash block chain. ZEC is the default (and currently the only defined) asset for the Zcash mainnet.
+- Zcash Shielded Asset: an asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard anonymity pool.
+- Wrapped Asset: a ZSA asset with native issuance defined outside the Zcash block chain.
+
+Abstract
+========
+
+ZIP 226 [#zip-0226]_ and ZIP 227 [#zip-0227]_ propose in conjunction the Zcash Shielded Assets (ZSA) protocol, which is an extension of the
+Orchard protocol that enables the creation, transfer and burn of custom assets on the Zcash chain. The creation of such assets is defined
+in ZIP 227 [#zip-0227]_. The transfer and burn of such assets is defined in ZIP 226 [#zip-0226]_.
+
+Motivation
+==========
+
+The current Orchard protocol does not support custom assets, which is an important aspect for the proper
+evolution of the Zcash ecosystem. Enabling multi-asset support on the Zcash chain will open the door for a host of
+applications and enhance the ecosystem with application developers and asset custody institutions for issuance and
+bridging purposes.
+
+Overview & Rationale
+====================
+
+This ZIP specifies the issuance mechanism of the Zcash Shielded Assets (ZSA) protocol and must work conjunction with ZIP 226 [#zip-0226]_, as the issuance mechanism is only valid for the ZSA transfer protocol, as it produces notes that can only be transferred under ZSA.
```

Comment:
```suggestion
This ZIP specifies the issuance mechanism of the Zcash Shielded Assets (ZSA) protocol. It must work in conjunction with ZIP 226 [#zip-0226]_, as the issuance mechanism is only valid for the ZSA transfer protocol, because it produces notes that can only be transferred under ZSA.
```

Reply:
Suggestion incorporated, with slight changes, in [PR#649](https://github.com/zcash/zips/pull/649).